resources/documentation/deploy_bluebanquise.rst

```diff
@@ -148,6 +148,18 @@ Note also that if you update configuration on management1, it is recommanded to
 
 Please refer to the pxe_stack role dedicated section in this documentation for more information on the bootset usage.
 
+SSH public key
+--------------
+
+In order to log into the remote nodes without giving the password, check that
+the ssh public key defined in authentication.yml in your inventory match your
+management1 public key. If not, update it. Remember to run the pxe_stack role
+after updating the configuration.
```

Comment:
I would have added the role ssh_slave, for those he uses it. No ?

Reply:
At this step of the documentation, we are deploying our first nodes in the cluster. We need to update the kickstart configuration (templates/kickstart.cfg.j2) to insert the new SSH public key, which will allow to authenticate to the deployed hosts without password authentication.
Running ssh_slave may add the public key into the authorized_keys file of management1. Since we already ran Ansible successfully, it should already be configured. Am I wrong?

Reply:
indeed, at this step it is useless to indicate this way.
Good for me.